tenacity/__init__.py

```diff
@@ -95,153 +83,32 @@ def wrapped_f(*args, **kw):
         return wrap
 
 
-class TryAgain(Exception):
-    """Always retry the executed function when raised."""
-
-
 class Retrying(object):
     """Retrying controller."""
 
-    def __init__(self,
-                 stop=stop_never, wait=wait_none(),
-                 sleep=time.sleep,
-                 retry=retry_if_exception_type(),
-                 before=before_nothing,
-                 after=after_nothing,
-                 reraise=False):
+    def __init__(self, marshaller, sleep=time.sleep):
```

Comment:
This is breaking the public API so this is not an option.

You should create the marshaller in `__init__` probably.